<launch>
    <!--Parameters-->
    <arg name="frame_id" default="map" />
    
    <!--node launch-->
    <node pkg= "decawave" ns="uwb" name="filtered" type="filter_UWBpoint_vicon" output="screen">
        <param name="frame_id" value="$(arg frame_id)" />
        <param name="uwbTopic" value="/uwb/point0/decaPoint" />
        <param name="viconTopic" value="/vicon/j8/pose" />

    </node>

</launch>
